.gitignore

```diff
@@ -7,3 +7,7 @@ __pycache__/
 voluncheer/db.sqlite3
 **/.coverage
 **/coverage.lcov
+.vscode/
```

Comment:
Please don't exclude the `.vscode` directory, we should keep the format and lint settings the same across our environments.